src/credential/mod.rs

```diff
@@ -203,10 +431,39 @@ pub enum CredentialFormat {
     #[serde(rename = "jwt_vc_json-ld")]
     JwtVcJsonLd,
     LdpVc,
+    #[serde(rename = "vcdm2_sd_jwt")]
+    SdJwt,
     #[serde(untagged)]
     Other(String), // For ease of expansion.
 }
 
+impl From<CredentialFormat> for ClaimFormatDesignation {
+    fn from(value: CredentialFormat) -> Self {
+        match value {
+            CredentialFormat::MsoMdoc => ClaimFormatDesignation::MsoMDoc,
+            CredentialFormat::JwtVcJson => ClaimFormatDesignation::JwtVcJson,
+            CredentialFormat::SdJwt | CredentialFormat::JwtVcJsonLd => {
+                ClaimFormatDesignation::Other(value.to_string())
+            }
+            CredentialFormat::LdpVc => ClaimFormatDesignation::LdpVc,
+            CredentialFormat::Other(s) => ClaimFormatDesignation::Other(s.to_owned()),
+        }
+    }
+}
```

Comment:
This gives me pause. A credential of one format could satisfy many claim format designations. Is this truly necessary, and where is it used?

Reply:
This is a type conversion to align with the `openid4vp` crate's `ClaimFormatDesignation` use in the descriptor map:

```rust
DescriptorMap::new(
    input_descriptor_id,
    ClaimFormatDesignation::from(credential.format()),
    "$.verifiableCredential".into(),
)
```

Reply:
commit [b783564](https://github.com/spruceid/mobile-sdk-rs/pull/43/commits/b7835647a2976b5a9dc82c9ba575d74050d54f3f) removes this conversion.